client/js/admin.ts

```diff
@@ -182,61 +185,134 @@ class ApplicantEntries {
 	}
 	private static updateFilter() {
 		this.filter = {};
+
 		if (this.branchFilter.value !== "*") {
-			this.filter.branch = this.branchFilter.value;
+			let [, type, branchName] = this.branchFilter.value.match(/^(application|confirmation)-(.*)$/)!;
+			if (type === "application") {
+				this.filter.application_branch = branchName;
+			}
+			else if (type === "confirmation") {
+				this.filter.confirmation_branch = branchName;
+			}
 		}
-		if (this.statusFilter.value !== "*") {
-			this.filter.status = this.statusFilter.value;
+
+		switch (this.statusFilter.value) {
+			case "no-decision":
+				this.filter.accepted = false;
+				break;
+			case "accepted":
+				this.filter.accepted = true;
+				break;
+			case "not-confirmed":
+				this.filter.accepted = true;
+				this.filter.attending = false;
+				break;
+			case "confirmed":
+				this.filter.accepted = true;
+				this.filter.attending = true;
+				break;
 		}
+
 		this.offset = 0;
 		this.load();
 	}
 	public static load() {
 		const status = document.getElementById("applicants-entries-status") as HTMLParagraphElement;
 		status.textContent = "Loading...";
 
-		let query: { [index: string]: any } = {
+		let query = `
+		query($search: String!, $useRegex: Boolean!, $offset: Int!, $count: Int!, $filter: UserFilter!) {
+			search_user(search: $search, use_regex: $useRegex, offset: $offset, n: $count, filter: $filter) {
+				offset,
+				count,
+				total,
+				users {
```

Comment:
@illegalprime Is there a way to just specify "Get all info about the user"? That would clean up this query a lot